doc/website/getting-started/installation.md

```diff
@@ -54,6 +53,16 @@ sudo apt install gcc g++ cmake libacl1-dev libncurses5-dev pkg-config
 
 Additionally, there is an optional dependency to the MIT licensed [cpptoml](https://github.com/skystrife/cpptoml) library, which is used to parse a RouDi config file for the mempool config.
 
+### QNX
+
+QNX SDP 7.0 and 7.1 are supported (shipping with gcc 5.4 and gcc 8.3 respectively).
+
+Easiest way to build iceoryx on QNX is using the build script and providing a toolchain file.<br>
+Example:
+```
+./tools/iceoryx_build_test.sh -t /home/user/toolchains/qnx/qcc_x86_64_toolchain.cmake
```

Comment:
could we add a toolchain file to iceoryx? Doesn't seem to be too complicated
https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html#cross-compiling-for-qnx
https://cristianadam.eu/20181202/a-better-qnx-cmake-toolchain-file/

Reply:
+1
Also see #609.